test/functional/rpc_net.py

```diff
@@ -6,6 +6,7 @@
 
 Tests correspond to code in rpc/net.cpp.
 """
+import time
```

Comment:
nit: You're only using `time` to set the time field in the ADDR message. As long as the `CAddress`'s time is greater than             100000000, it'll be accepted. You can avoid this import by just setting time to `100000000 + i` or similar